content/post/2023/ilugc-meet-january-2023.md

```diff
@@ -0,0 +1,45 @@
+---
+title: ILUGC Monthly Meet - January 21, 2023 - 3.00 PM IST
+date: 2023-01-20T20:30:00+05:30
+url: /ilugc-meet-january-2023
+author: Parameshwar Arunachalam 
+---
+
+Indian Linux Users Group, Chennai [ ILUGC ] has been spreading
+awareness on Free/Open Source Software (F/OSS) in Chennai since
+January 1998.
+
+We usually meet on the second Saaturday of every month, and for the
+month of **May**, we shall meet through Jitsi on **Saturday 21, 2023 at 1500
```

Comment:
`**May**` need to be changed to `**January**`

Reply:
Changing now,
Sorry for the typos

Reply:
Updated